### PR TITLE
BDLaethb3

### PR DESCRIPTION
**Pull request by thea-m**
Happy new year! :)
Here is the next manuscript. I think that this one could benefit from someone with better Amharic than me looking at the text of the additions, which mention interesting historical figures (maybe at a later stage of our workflow). 
I look forward to your comments!

### CODE REVIEW

File: OxfordBodleian/Juel-Jensen/BDLaethb3.xml [hunk 1]
```diff
@@ -189,29 +758,284 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
                                         <dim type="intercolumn">26</dim>
                                     </dimensions>
+                                    <ab type="pricking">Ruling and pricking are visible. Both primary and ruling pricks are visible.</ab>
+                                    <ab type="ruling" subtype="pattern">Ruling pattern: 1A-1A-1A1A/0-0/0-0/C.</ab>
+                                    <ab type="ruling"> The upper line is written above the ruling. The bottom line is written above the ruling.</ab>
+                                </layout>
+                                
+                                <layout columns="3" writtenLines="27">
+                                    <locus from="5ra" to="11rb"/>
+                                    <!--<note corresp="#textarea #margin">Data on text area and margin dimensions taken from <locus target="#"/>.</note>-->
```
Comment (MassimoVilla) [inline OxfordBodleian/Juel-Jensen/BDLaethb3.xml:794]: This info has not been taken? It should be added on the to-do agenda

File: OxfordBodleian/Juel-Jensen/BDLaethb3.xml [hunk 2]
```diff
@@ -189,29 +758,284 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                                    numerals and elements of numerals and punctuation signs.</seg>
+                            </handNote>
+                        </handDesc>
+                        
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus from="1va" to="1vb"/>
+                                    <desc type="LandGrant">Record in Gǝʿǝz and Amharic by <persName ref="PRS5636IyoasI"/> and <persName ref="PRS2682BerhanM"/>, granting lands to 
+                                        daǧāzmāčč ʾƎšate, probably to be identified with <persName ref="PRS3856esateof"/>, in recognition of his deeds in war. The record mentions ‘the children of
```
Comment (MassimoVilla) [inline OxfordBodleian/Juel-Jensen/BDLaethb3.xml:897]: EAE II, p. 62a says "daǧǧāzmāčč". Is the spelling form "daǧāzmāčč" also good?

File: OxfordBodleian/Juel-Jensen/BDLaethb3.xml [hunk 3]
```diff
@@ -189,29 +758,284 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                                    </q>
+                                </item>
+                                
+                                <item xml:id="a5">
+                                    <locus target="#11v #96r"/>
+                                    <desc type="StampExlibris">Two stamps, hardly legible, probably by <persName ref="PRS10450YosabI"/>, whose name
+                                        (<foreign xml:lang="ar">ʾAnbā</foreign> Yūsāb) is given next to them. Between the stamps is written, from right to left, on 
+                                        <locus target="#11v"/>:
+                                    <foreign xml:lang="ar">انبا يوساب</foreign>, <foreign xml:lang="ar">باسم الله الرووف الرحيم المجد لله في العلا</foreign>,
+                                        and <foreign xml:lang="ar">حَرمة</foreign>, and on <locus target="#96r"/> <foreign xml:lang="ar">باسم الله الرووف الرحيم المجد لله في العلا</foreign> .</desc>
```
Comment (MassimoVilla) [inline OxfordBodleian/Juel-Jensen/BDLaethb3.xml:965]: Is there an alif in the first word? I think I read بسم (with alif omitted in a very common word, like hādā هذا) but I might be wrong.
Sorry, I couldn't find fol. 11v in the photo gallery so I couldn't check the first Arabic note

**Review by MassimoVilla — CHANGES_REQUESTED** [review]
Thank you for this intriguing MS. I have not gone in depth into the notes in Amharic, I think it will be good to involve someone with a better competence than mine.

File: OxfordBodleian/Juel-Jensen/BDLaethb3.xml (hunk 1 above)
Comment (thea-m) [reply OxfordBodleian/Juel-Jensen/BDLaethb3.xml:794]: already on the list with requests for further actions :)

**Comment by thea-m** [comment]
Thank you, Massimo!

**Review by MassimoVilla — APPROVED** [review]

File: OxfordBodleian/Juel-Jensen/BDLaethb3.xml [hunk 4]
```diff
@@ -33,15 +33,583 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                    </msContents>
+                    
 
                     <physDesc>
                         <objectDesc form="Codex">
                             <supportDesc>
                                 <support>
                                     <material key="parchment"/>
+                                    <p><locus target="#14"/> is cut off after the first column. Very rare parchment repairs (e.g. <locus target="#202"/>), presumably from the 
+                                        parchmenting process.
```
Comment (ElianaDalSasso) [inline OxfordBodleian/Juel-Jensen/BDLaethb3.xml:609]: I think that in the end we will have to check all the formulas used to indicate the process. At the moment they vary from parchmenting process, parchment making, and parchment making process. For me they are all synonyms but perhaps among these there is a more formal one.

**Review by ElianaDalSasso — APPROVED** [review]
Great job Thea! I had just a minor comment which however pertains to the general final review.

File: OxfordBodleian/Juel-Jensen/BDLaethb3.xml (hunk 4 above)
Comment (thea-m) [reply OxfordBodleian/Juel-Jensen/BDLaethb3.xml:609]: Absolutely, we need to do this category by category

Comment (gnizla) [reply OxfordBodleian/Juel-Jensen/BDLaethb3.xml:609]: Thanks, if I understood this line of conversation, I don't think we need to standardize everything we say: I think it is fine to use synonyms.

**Pull request merged**